src/translation/translation_fr_FR.ts

```diff
@@ -2515,7 +2515,7 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../connectdlg.cpp" line="75"/>
         <source>The Connection Setup window lists the available servers registered with the selected directory. Use the Directory dropdown to change the directory, find the server you want to join in the server list, click on it, and then click the Connect button to connect. Alternatively, double click on the server name to connect.</source>
-        <translation>La fenêtre de configuration de la connexion répertorie les serveurs disponibles enregistrés dans l&apos;annuaire sélectionné. Utilisez la liste déroulante Annuaires pour changer de serveur annuaire, trouvez le serveur que vous voulez rejoindre dans la liste, cliquez dessus, puis cliquez sur le bouton Connect pour vous connecter. Vous pouvez également double-cliquer sur le nom du serveur pour vous connecter.</translation>
+        <translation>La fenêtre de configuration de la connexion répertorie les serveurs disponibles enregistrés dans l&apos;annuaire sélectionné. Utilisez la liste déroulante Annuaires pour changer d&apos;annuaire, trouvez le serveur que vous voulez rejoindre dans la liste, cliquez dessus, puis cliquez sur le bouton Se connecter pour vous connecter. Vous pouvez également double-cliquer sur le nom du serveur pour vous connecter.</translation>
```

Comment:
Mm, here the UI label in English is `Directory:`.  I've not checked the French but `la liste déroulante Annuaires` would then need to use the singlular.

Reply:
(And yes, we could do with a tool that makes this kind of cross-referencing ... not a problem.)